packages/cli/src/lib/autorun/autorun-command.unit.test.ts

```diff
@@ -74,4 +78,18 @@ describe('autorun-command', () => {
       },
     } satisfies PortalUploadArgs);
   });
+
+  it('should call collect and upload and ensure json format', async () => {
+    await cli({ 'persist.format': ['md'] }).parseAsync();
+
+    expect(collectAndPersistReports).toHaveBeenCalledWith(
+      expect.objectContaining({
+        verbose: true,
+        config: '/test/code-pushup.config.ts',
+        persist: expect.objectContaining({
+          format: ['md', 'json'],
+        }),
+      }),
+    );
+  });
```

Comment:
```suggestion
```

This logic is handled by the config-middleware, if I understand it correctly. So once a separate test case is added there for the addition of the `json` format (as commented above), it is unnecessary to create duplicate tests for this case elsewhere.